### PR TITLE
[SP-6463] Remove cancelled events

### PR DESCRIPTION
**Pull request by paweldonath-silvair**

### CODE REVIEW

File: test/signal_level_unittest.py [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+        print((40 * '='))
+
+    def tearDown(self):
+        self.adr_client.event_handler.update_all_events({}, '')  # Clear out the database
+
+    def test_active_event_with_single_interval(self):
+        print('in test_active_event_with_single_interval()')
+        events = [self.event_active_single_interval]
+
+        with freeze_time("2020-03-18 07:00:00"):
```
Comment (bartoszflis-silvair) [inline test/signal_level_unittest.py]: I guess that time is somehow related to time of the event. Perhaps there's a way to express it here?

**Review by bartoszflis-silvair — APPROVED** [review]

File: test/signal_level_unittest.py (hunk 1 above)
Comment (bartoszflis-silvair) [reply test/signal_level_unittest.py]: Bump.

Comment (bartoszflis-silvair) [inline test/signal_level_unittest.py]: I guess that time is somehow related to time of the event. Perhaps there's a way to express it here?

**Review by bartoszflis-silvair — CHANGES_REQUESTED** [review]

File: test/signal_level_unittest.py (hunk 1 above)
Comment (paweldonath-silvair) [reply test/signal_level_unittest.py]: In a method `setUp` all events are read from XML and I wrote a comment with summary. Would it be more readable if I repeated this summary in every test?

**Review by bartoszflis-silvair — APPROVED** [review]

**Review by bartoszflis-silvair — APPROVED** [review]

**Pull request merged**